### ISSUE TITLE
Error when passing keyword arguments to the add_image method

### ISSUE THREAD

**Issue by georgebv** [SciTools/cartopy#1567]
### Description
I am getting an exception when passing keyword arguments like `zorder` or `alpha` to the `.add_image` method of a `GeoAxesSubplot` object. It renders fine with zorder and alpha as expected, but seconds later it throws an exception.

#### Code to reproduce

```
fig, ax = plt.subplots(subplot_kw=dict(projection=cartopy.crs.PlateCarree()))
ax.add_image(cartopy.io.img_tiles.GoogleTiles(url=r'https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}'), 1, zorder=5, alpha=0.5)
```

#### Traceback 

```
Traceback (most recent call last):
  File "C:\Users\grbh\Anaconda3\envs\py38\lib\site-packages\matplotlib\backends\backend_qt5.py", line 498, in _draw_idle
    self.draw()
  File "C:\Users\grbh\Anaconda3\envs\py38\lib\site-packages\matplotlib\backends\backend_agg.py", line 393, in draw
    self.figure.draw(self.renderer)
  File "C:\Users\grbh\Anaconda3\envs\py38\lib\site-packages\matplotlib\artist.py", line 38, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
  File "C:\Users\grbh\Anaconda3\envs\py38\lib\site-packages\matplotlib\figure.py", line 1735, in draw
    mimage._draw_list_compositing_images(
  File "C:\Users\grbh\Anaconda3\envs\py38\lib\site-packages\matplotlib\image.py", line 137, in _draw_list_compositing_images
    a.draw(renderer)
  File "C:\Users\grbh\Anaconda3\envs\py38\lib\site-packages\matplotlib\artist.py", line 38, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
  File "C:\Users\grbh\Anaconda3\envs\py38\lib\site-packages\cartopy\mpl\geoaxes.py", line 479, in draw
    return matplotlib.axes.Axes.draw(self, renderer=renderer, **kwargs)
  File "C:\Users\grbh\Anaconda3\envs\py38\lib\site-packages\matplotlib\artist.py", line 38, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
TypeError: draw() got an unexpected keyword argument 'zorder'
```

<details>
 <summary>Full environment definition</summary>

### Operating system
Windows 10

### Cartopy version
0.18.0

### conda list

```
```

### pip list

```
```

</details>

**Comment by greglucas** [comment]
This should be fixed by #1565.

**Comment by QuLogic** [comment]
Closing as duplicate of #1563.

**Comment by feliperiosg** [comment]
the bugs persists as of October 7, 2020

**Comment by Dapid** [comment]
The problem persists on Cartopy master.

**Comment by greglucas** [comment]
Yes, the fix in #1565 was targeted at branch `0.18.x` If you need source for the fix, you can build from that branch: https://github.com/SciTools/cartopy/tree/v0.18.x
